app/models/concerns/blacklight/marc/document_extension.rb

```diff
@@ -0,0 +1,214 @@
+# frozen_string_literal: true
```

Comment:
This is the new path for app/models/concerns/solr/document/marc.rb.  Git isn't recognizing it as the same file, though.